BedrockServer.h

```diff
@@ -245,7 +253,7 @@ class BedrockServer : public SQLiteServer {
     const SData args;
 
     // This is the thread that handles a new socket, parses a command, and queues it for work.
-    void handleSocket(Socket&& s, bool isControlPort);
+    void handleSocket(Socket&& socket, bool fromControlPort, bool fromPublicCommandPort, bool fromPrivateCommandPort);
```

Comment:
Maybe add some doc explaining what's `fromPublicCommandPort` vs `fromPrivateCommandPort`? Not obvious to me

Reply:
Adding in the .h file.

Reply:
Oh, this is the .h file.